build.sbt

```diff
@@ -46,7 +46,7 @@ OsgiKeys.additionalHeaders := Map("-removeheaders" -> "Include-Resource,Private-
 // publishing
 ///////////////
 
-crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0")
+crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.1")
```

Comment:
It's great that specs and scalacheck come with a version that cross-compiles for all three Scala version.